pulp_docker/app/registry.py

```diff
@@ -211,3 +219,160 @@ def _base_paths(path):
                                                 headers)
             else:
                 return await self._stream_content_artifact(request, web.StreamResponse(), ca)
+
+    @staticmethod
+    def verify_token(request, access_action):
+        """Verify a Bearer token."""
+        token_verifier = TokenVerifier(request, access_action)
+        token_verifier.verify()
+
+
+class TokenVerifier:
+    """A class used for a token verification."""
+
+    def __init__(self, request, access_action):
+        """
+        Store data required for the token verification.
+
+        Args:
+            request (:class:`~aiohttp.web.Request`): The request with an Authorization header.
+            access_action (str): A required action to perform pulling/pushing.
+
+        """
+        self.request = request
+        self.access_action = access_action
+
+    def verify(self):
+        """Verify a Bearer token."""
+        authorization = self.get_authorization_header()
+        self.check_authorization_token(authorization)
+
+    def get_authorization_header(self):
+        """
+        Fetch an Authorization header from the request.
+
+        Raises:
+            web.HTTPUnauthorized: An Authorization header is missing in the header.
+
+        Returns:
+            A raw string containing a Bearer token.
+
+        """
+        try:
+            return self.request.headers['Authorization']
+        except KeyError:
+            authenticate_header = self._build_authenticate_header()
+            raise web.HTTPUnauthorized(
+                headers={'Www-Authenticate': authenticate_header},
+                reason='Access to the requested resource is not authorized. '
+                       'A Bearer token is missing in a request header.'
```

Comment:
i think docs and the code is a bit in disagreement. https://github.com/pulp/pulp_docker/pull/412/files#diff-76c460da6b178be6b7501110986f861bR11

disabled by default means - it is not enabled and everything will still works. In our case pulp still expects a token and it will fail
shall we re-phrase the docs and say - token auth is not configured by default, please provide x,y,z setting and variables?